app/config/config.yml

```diff
@@ -71,3 +72,28 @@ swiftmailer:
     spool:     { type: memory }
 
 fos_http_cache: ~
+
+fos_rest:
+    routing_loader:
+        default_format: json
+    view:
+        formats:
+            json: true
+        view_response_listener: 'force'
+    format_listener:
+        rules:
+            - { path: '^/api', priorities: ['json'], fallback_format: json, prefer_extension: true }
+            - { path: '^/', stop: true }
+    exception:
+        codes:
+            'Symfony\Component\Routing\Exception\ResourceNotFoundException': 404
+        messages:
+            'Symfony\Component\Routing\Exception\ResourceNotFoundException': true
+
+sensio_framework_extra:
+    view:    { annotations: false }
+
+swp_updater:
+    version_class: %swp_updater.version.class%
+    client:
+        base_uri: http://updates.sourcefabric.org
```

Comment:
yeah its base uri of the updates server, but in the context that we have the structure of clients in the UpdaterBundle.